packages/pds/src/db/migrations/20221021T162202001Z-init.ts

```diff
@@ -77,10 +83,18 @@ export async function up(db: Kysely<unknown>, dialect: Dialect): Promise<void> {
     .addColumn('did', 'varchar', (col) => col.notNull())
     .addColumn('collection', 'varchar', (col) => col.notNull())
     .addColumn('rkey', 'varchar', (col) => col.notNull())
-    .addColumn('raw', 'text', (col) => col.notNull())
     .addColumn('receivedAt', 'varchar', (col) => col.notNull())
     .addColumn('indexedAt', 'varchar', (col) => col.notNull())
     .execute()
+  // Ipld Blocks
+  await db.schema
+    .createTable(ipldBlockTable)
+    .addColumn('cid', 'varchar', (col) => col.primaryKey())
+    .addColumn('did', 'varchar', (col) => col.notNull())
```

Comment:
If two dids attempt to store the same cid, could we get into trouble?  Or perhaps there's some reason that is guaranteed not to happen.

Reply:
Unless there's a very clear reason why that shouldn't happen, I might suggest making this a compound key `(did, cid)`, and then add an index on `cid`.  Or if we want to avoid the possible duplicate storage of `content`, breaking the did relationship out into a separate table.

Reply:
ugh you're right...

I'd like for `cid` to be the primary key here, it just sits better with me. So I think i'll break out the did-cid relationship 🤔

Reply:
Sounds like a plan 👍